stablehlo/dialect/VhloBase.td

```diff
@@ -34,6 +34,49 @@ def VersionedInterface : OpInterface<"VersionedInterface"> {
   ];
 }
 
+def VHLO_VersionedAttrInterface : AttrInterface<"VersionedAttrInterface"> {
+  let cppNamespace = "::mlir::vhlo";
+  let methods = [
+    InterfaceMethod<
+      "Returns the minimum version an attribute is supported in.",
```

Comment:
How about expanding to "the minimum version of the VHLO dialect"? (Here, below and above).